examples/browser/extension/utils/build.js

```diff
@@ -1,9 +1,9 @@
-var webpack = require("webpack"),
-    config = require("../webpack.config");
+var webpack = require('webpack')
+var config = require('../webpack.config')
```

Comment:
why `var` over `const`? maybe the linter should notice?

Reply:
Probably the linter assumes that it is ES5 code. I would like to revisit examples later.